test/run

```diff
@@ -127,9 +127,10 @@ test_docker_run_usage() {
 test_scl_usage() {
   local run_cmd="$1"
   local expected="$2"
+  local cid_file="$CID_FILE_DIR/$(mktemp -u -p . --suffix .cid)"
 
   info "Testing the image SCL enable"
-  out=$(docker run --rm ${IMAGE_NAME} /bin/bash -c "${run_cmd} 2>&1")
+  out=$(docker run --cidfile ${cid_file} ${IMAGE_NAME} /bin/bash -c "${run_cmd} 2>&1")
```

Comment:
option --rm should be there. As soon as container is exited, then container can be deleted.

Reply:
Oh sorry, I thought I removed this commit.